services/provider/server/server_test.go

```diff
@@ -396,8 +396,8 @@ func TestGetExternalResources(t *testing.T) {
 	// When ocsv1alpha1.StorageConsumerStateReady but ceph resources is empty
 	req.StorageConsumerUUID = string(consumerResource5.UID)
 	storageConRes, err = server.GetStorageConfig(ctx, &req)
-	assert.Error(t, err)
-	assert.Nil(t, storageConRes)
+	assert.NoError(t, err)
+	assert.NotNil(t, storageConRes)
```

Comment:
we should not have these changes, please remove secrets from this file related to monitor and health checker

Reply:
Tracking in this https://github.com/red-hat-storage/ocs-operator/issues/2829